responder/config_sync.go

```diff
@@ -0,0 +1,83 @@
+package responder
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/flanksource/commons/logger"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+)
+
+func StartConfigSync() {
+	for {
+		logger.Infof("Syncing configuration")
+		syncConfig()
+
+		time.Sleep(10 * time.Minute)
```

Comment:
Is 10 minutes good enough ?

Reply:
Its probably too often - I think change to 1h, and then when we build the integration table we can add a schedule